### ISSUE TITLE
Can't install Google Apps

### ISSUE THREAD

**Issue by ane-gabriela** [xwikisas/application-googleapps#26]
Steps to reproduce:

1. Access a cloud instance or a local one
2. Install the Google App 2.4

Expected results: The app is installed.

Actual results: The app can't be installed, compatibility issues appear.

Environment: XWiki 10.11.9 and 8.4.5 with Chrome 76

![8 4 5](https://user-images.githubusercontent.com/40691112/64164841-9722fe00-ce4c-11e9-9309-da4fd4f6d36b.jpg)
![10 11 9](https://user-images.githubusercontent.com/40691112/64164846-99855800-ce4c-11e9-97ad-93757087a597.jpg)

https://up1.xwikisas.com/#ufEzF4DyF7GO4QGoOveAWg

**Comment by mflorea** [comment]
Regression caused by 68af57f5e83eea020a2aee34f64e8b3f1efc4491 .

EM resolves the dependency tree differently than Maven does. With Maven you get this https://up1.xwikisas.com/#1IMvC6cxq47GFEQG1181CQ . If you look for ``org.apache.httpcomponents:httpclient`` you'll see it is resolved to version 4.5.2. **But this is not right at runtime**. If you try to install the app with EM you get this https://up1.xwikisas.com/#ufEzF4DyF7GO4QGoOveAWg . EM resolves ``org.apache.httpcomponents:httpclient`` to version 4.5.9. There are two paths in the dependency tree that lead to ``org.apache.httpcomponents:httpclient``. See https://up1.xwikisas.com/#HiFRTCUmfZYYWoIT72kwXQ . google-http-client https://search.maven.org/artifact/com.google.http-client/google-http-client/1.30.1/jar doen't specify the version of httpclient and its parent pom has this https://search.maven.org/artifact/com.google.http-client/google-http-client-parent/1.30.1/pom

```
<dependency>
  <groupId>org.apache.httpcomponents</groupId>
  <artifactId>httpclient</artifactId>
  <version>${project.httpclient.version}</version>
</dependency>
...
<project.httpclient.version>4.5.8</project.httpclient.version>
```

It's not clear why EM wants 4.5.9 and not 4.5.8 but it doesn't matter. The fact is that ``google-api-client`` 1.30.2 has a transitive dependency on ``httpclient`` 4.5.8 which is not compatible with the 4.5.2 version available in XWiki 8.4. ATM we can only upgrade to something that uses ``google-http-client`` 1.23.0, see https://search.maven.org/artifact/com.google.http-client/google-http-client-parent/1.23.0/pom .

Note that httpclient 4.5.9 is available in XWiki only since 11.5RC1, and 4.5.8 since 11.3RC1, so we won't be able to upgrade to the latest ``google-http-client`` too soon..

**Comment by mflorea** [comment]
@tmortagne tells me that the enforcer plugin would have helped us catching this type of issues at build time, but this check is enabled since XWiki 9.0RC1. So until we upgrade our parent to 9.x we need to test at runtime with EM the dependency upgrades we do.
